controller/app.controller.js

```diff
@@ -22,4 +22,23 @@ exports.getAllEndpoints = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params
+
+    if (article_id > 13){
+        return res.status(404).send({message: "please provide a valid numerical id path"})
+    }
```

Comment:
While this may work in your test envrioment as soon as this gos to production and there is more than 13 articles this will cause a big problem.
If all we're testing for is that it is a valid number, take a look at what PSQL returns if we input anything other than a number into the model.